.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: node_js
+
+notifications:
+  email:
+    on_success: never
+
+jobs:
+  include:
+    - stage: "PHP"
+      name: "PHPStan on PS nightly and unit tests"
+      language: php
+      php: 7.2
+      script:
+        - composer install
+        # PHP syntax check
+        - bash -c '! (find . -name "*.php" ! -path "./vendor/*" -print0 | xargs -0 -n1 -P4 php -l | grep "Parse error")'
+        # PHP cs fixer
+        - composer run lint
+        # PHP Unit
+        # - composer run test
+        # PHP Stan
+        - docker run -tid --rm -v ps-volume:/var/www/html --name temp-ps prestashop/prestashop:nightly
```

Comment:
Whatch out, I update facetedsearch template to prevent a bug :D

Reply:
Thanks (Link https://github.com/PrestaShop/ps_facetedsearch/commit/20db127bc626d12660db57e7e5b591a47f0a43fd#diff-354f30a63fb0907d4ad57269548329e3)